sdk/eas/README.md

```diff
@@ -1,187 +1,389 @@
+- ✅ **Multi-Operations**: Support for batch attestations and revocations
+- ✅ **Validation**: Built-in attestation validation and timestamp queries
+
+## Installation
+
+```bash
+npm install @settlemint/sdk-eas
+# or
+yarn add @settlemint/sdk-eas
+# or
+bun add @settlemint/sdk-eas
+```
+
+## Quick Start
+
+```typescript
+import { createEASClient } from "@settlemint/sdk-eas";
+
+// Initialize EAS client with Portal credentials
+const client = createEASClient({
+  instance: "https://attestation-portal-ee231.gke-europe.settlemint.com/graphql",
+  accessToken: "sm_aat_your_access_token",
+  debug: true,
+});
+
+// Deploy EAS contracts (if needed)
+const deployment = await client.deploy();
+console.log("EAS deployed at:", deployment.easAddress);
+console.log("Schema Registry at:", deployment.schemaRegistryAddress);
+
+// Register a schema
+const schemaResult = await client.registerSchema({
+  fields: [
+    { name: "user", type: "address", description: "User's wallet address" },
+    { name: "score", type: "uint256", description: "Reputation score" },
+    { name: "verified", type: "bool", description: "Verification status" },
+  ],
+  resolver: "0x0000000000000000000000000000000000000000",
+  revocable: true,
+});
+
+// Create an attestation
+const attestationResult = await client.attest({
+  schema: schemaResult.hash,
+  data: {
+    recipient: "0x742d35Cc6634C0532925a3b8D4C9db96C4b4d8b6",
+    expirationTime: BigInt(0),
+    revocable: true,
+    refUID: "0x0000000000000000000000000000000000000000000000000000000000000000",
+    data: "0x", // Encoded attestation data
+    value: BigInt(0),
+  },
+});
+
+console.log("Attestation created:", attestationResult.hash);
+```
+
+## Portal GraphQL Integration
+
+This package integrates with SettleMint's Portal GraphQL API, providing access to all EAS operations:
+
+### Available Mutations
+
+- `EasSchemaRegistryRegister` - Register new schemas
+- `EasDeploymentAttest` - Create single attestation
+- `EasDeploymentAttestByDelegation` - Create delegated attestation
+- `EasDeploymentMultiAttest` - Create multiple attestations
+- `EasDeploymentMultiAttestByDelegation` - Create multiple delegated attestations
+- `EasDeploymentRevoke` - Revoke single attestation
+- `EasDeploymentRevokeByDelegation` - Revoke by delegation
+- `EasDeploymentMultiRevoke` - Revoke multiple attestations
+- `EasDeploymentMultiRevokeByDelegation` - Revoke multiple by delegation
+- `EasDeploymentRevokeOffchain` - Revoke offchain attestation
+- `EasDeploymentMultiRevokeOffchain` - Revoke multiple offchain
+- `EasDeploymentTimestamp` - Timestamp operation
+- `EasDeploymentMultiTimestamp` - Timestamp multiple operations
+- `EasDeploymentIncreaseNonce` - Increase nonce for delegations
+
+### Available Queries
+
+- Contract deployment status and addresses
+- Schema retrieval and validation
+- Attestation lookup and verification
+- Transaction monitoring and history
 
 ## API Reference
 
-### Functions
+### EASClient
```

Comment:
Typedocs will generate most of these, no need to do this manually. Make sure to have good jsDoc comments on the EASClient with '@example' also